cdap-master/src/main/java/co/cask/cdap/data/tools/DataMigration.java

```diff
@@ -113,7 +121,11 @@ protected void configure() {
 
       switch (action) {
         case METRICS:
-          migrateMetricsData(injector);
+          boolean keepOldTable = false;
```

Comment:
Before switch - remove action from args. So that at this point you have only args for specific action. Then, here:
* if > 1 argument - tell that invalid arguments, and print help
* if 1 argument - check that is it the KEEP_OLD_METRICS_TABLES, otherwise - tell invalid args, print help
* if 0 args - use default KEEP_OLD_METRICS_TABLES